packages/grid/_modules_/grid/hooks/root/index.ts

```diff
@@ -1,5 +1,5 @@
 export * from './useApi';
-export * from './useGridApiEventHandler';
+export type { GridCallbackDetails } from './useGridApiEventHandler';
```

Comment:
Where the potential BC is. I don't think that we should document it as a BC in the changelog because we don't document what's exported here.

Reply:
I'll remove all those `export *` for the hooks at one point to stop exporting the hooks we do not want to expose.
I don't think stop exporting `useGridApiEventHandler` is an issue